runtime/build.gradle.kts

```diff
@@ -153,10 +153,25 @@ targetList.forEach { targetName ->
         includeRuntime()
     }
 
+    createTestTask(
+            project,
+            "ExperimentalMMStdAlloc",
+            "${targetName}ExperimentalMMStdAllocRuntimeTests",
+            listOf(
+                "${targetName}Runtime",
+                "${targetName}ExperimentalMemoryManager",
+                "${targetName}Release",
```

Comment:
(Probably unrelated to this PR, just curious.) Why `release`?

Reply:
Didn't want to collect all the possible combinations at the get go. But it seems I might have to include Debug/Release in the near future.